src/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp

```diff
@@ -437,17 +438,13 @@ std::vector<ov::Tensor> SubgraphBaseTest::calculate_refs() {
     update_ref_model();
     match_parameters();
```

Comment:
Please, do not forget about `match_parameters`. It cam be applicable for other tests. Just check the above 2 functions in details :)